src/Automation/TargetElementLocator.cs

```diff
@@ -1,29 +1,41 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using Axe.Windows.Actions.Contexts;
+using Axe.Windows.Automation.Resources;
 using Axe.Windows.Core.Bases;
 using Axe.Windows.Desktop.UIAutomation;
 using System;
+using System.Collections.Generic;
 using System.Globalization;
+using System.Linq;
 
 namespace Axe.Windows.Automation
 {
     class TargetElementLocator : ITargetElementLocator
     {
-        public A11yElement LocateRootElement(int processId)
+        public IEnumerable<A11yElement> LocateRootElements(int processId)
         {
             try
             {
-                var element = A11yAutomation.ElementFromProcessId(processId);
-
-#pragma warning disable CA2000 // Call IDisposable.Dispose()
-                return new ElementContext(element).Element;
-#pragma warning restore CA2000
+                var desktopElements = A11yAutomation.ElementsFromProcessId(processId);
+                return GetA11YElementsFromDesktopElements(desktopElements);
             }
             catch (Exception ex)
             {
                 throw new AxeWindowsAutomationException(string.Format(CultureInfo.InvariantCulture, DisplayStrings.ErrorFailToGetRootElementOfProcess, processId, ex), ex);
```

Comment:
Should this error be renamed from `DisplayStrings.ErrorFailToGetRootElementOfProcess` to something that reflects the potential for multiple elements?

Reply:
Suitably renamed